aspnetcore/migration/22-to-30.md

```diff
@@ -1025,7 +1025,13 @@ The following code shows the ASP.NET Core 2.2 template-generated `Program` class
 
 <xref:Microsoft.AspNetCore.Hosting.IWebHostBuilder> remains in 3.0 and is the type of the `webBuilder` seen in the preceding code sample. <xref:Microsoft.AspNetCore.Hosting.WebHostBuilder> will be deprecated in a future release and replaced by `HostBuilder`.
 
-The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). When using `HostBuilder`, you can only inject <xref:Microsoft.Extensions.Configuration.IConfiguration> and <xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment> into `Startup`'s constructor. The `HostBuilder` DI constraints:
+The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). When using `HostBuilder`, you can only inject the following into `Startup`'s constructor:
+
+* <xref:Microsoft.Extensions.Configuration.IConfiguration>
+* `Microsoft.Extensions.Hosting.IHostEnvironment`
```

Comment:
```suggestion
* <xref:Microsoft.Extensions.Hosting.IHostEnvironment>
```

Reply:
The xref seems to work when used with the `Extensions` namespace. I checked this on my machine using docfx, but I dunno if there's more to it than that.

Reply:
Can't do that yet ... no 3.0 API yet ... they're working on it.

Reply:
Just for completeness, this is where it takes me: https://docs.microsoft.com/en-gb/dotnet/api/microsoft.extensions.hosting.ihostenvironment?view=dotnet-plat-ext-3.0.

Reply:
Try this tho ... https://docs.microsoft.com/en-us/dotnet/api/?term=Microsoft.Extensions.Hosting.IHostEnvironment

Reply:
My understanding as of yesterday is that there's more work to be done in docs engineering on it. We have an open issue to update API links when we know that they've given the word that it's ready. Until then, I think the code fence is ok.

Reply:
Yeah, I understand. I provided the link just so you could see what I saw, but I never doubted that you knew better than I did. I just wondered if you'd missed the xref after changing the namespace. Sorry to have distracted you from your Azure AD party. :smiley:

Reply:
Azure AD Party 😄😢😄😠😄 ..... they must know that Halloween is here and that I needed a good *SCARE* today. 👻🎃

I have half of it working ... Managed Identities :+1: ... but Azure AD with a cert :-1:. ei ei ei ... I'll figure it out ... beat it into submission.

.... hummmmmm 🤔 ..........

I wonder if there are any open positions over at **_Apple docs_**??!!! :smile: